docs/configuration/multipledomains.md

```diff
@@ -18,14 +18,20 @@ Multiple top-level domains may also be setup by following the instructions below
         labels:
           - traefik.http.routers.${WARDEN_ENV_NAME}-varnish.rule=
               HostRegexp(`{subdomain:.+}.${TRAEFIK_DOMAIN}`)
+              || HostRegexp(`${TRAEFIK_DOMAIN}`)
```

Comment:
The lines adding the domain without the sub-domain match can use the shorter Host matcher:

```
Host(`${TRAEFIK_DOMAIN}`)
```

Could you update these?

Reply:
Updated